### PR TITLE
category

### PR DESCRIPTION
**Pull request by alethanhson**
https://trello.com/c/vqB47OnI/8-crud-category

image

### CODE REVIEW

File: resources/frontend/src/components/pages/Category/index.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            });
+        }
+    },
+    computed: {
+        ...mapGetters({
+            listCategory: 'Category/getCategories',
+        })
+    },
+}
+</script>
```
Comment (VanTan291) [inline resources/frontend/src/components/pages/Category/index.vue]: .

File: resources/frontend/src/store/Category/actions.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        return await api.post('/categories', formData).then((response) => {
+            if (response && response != undefined) {
+                return response
+            }
+        }).catch((error) => {
+            console.log(error);
+            return false
+        });
+    }
+}
```
Comment (VanTan291) [inline resources/frontend/src/store/Category/actions.js]: .

File: resources/frontend/src/store/Category/getters.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import state from "./state";
+
+export default {
+    getCategories: state => state.categories,
+}
```
Comment (VanTan291) [inline resources/frontend/src/store/Category/getters.js]: .

File: resources/frontend/src/store/Category/mutations.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import states from "./state";
+
+export default {
+    CATEGORIES(states, categories) {
+        states.categories = categories;
+    }
+}
```
Comment (VanTan291) [inline resources/frontend/src/store/Category/mutations.js]: .

File: resources/frontend/src/store/Category/state.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export default {
+    categories: [],
+}
```
Comment (VanTan291) [inline resources/frontend/src/store/Category/state.js]: .

File: resources/frontend/src/assets/css/style.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+.rounded-cs {
+    border-radius: 0.75rem;
+}
+
+.f-item {
+    width: 60px;
+}
```
Comment (VanTan291) [inline resources/frontend/src/assets/css/style.css]: .

File: resources/frontend/src/components/pages/Category/index.vue [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+        ...mapActions({
+            getListCategories: 'Category/getCategories',
+            storeCategory: 'Category/storeCategory',
+        }),
+
+        createCategory() {
+            this.storeCategory(this.category).then(result => {
+                if(result.data.code == 200) {
+                    this.getListCategories();
+                }else{
```
Comment (VanTan291) [inline resources/frontend/src/components/pages/Category/index.vue]: .

File: resources/frontend/src/components/pages/Category/index.vue [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+    methods: {
+        ...mapActions({
+            getListCategories: 'Category/getCategories',
+            storeCategory: 'Category/storeCategory',
+        }),
+
+        createCategory() {
+            this.storeCategory(this.category).then(result => {
+                if(result.data.code == 200) {
```
Comment (VanTan291) [inline resources/frontend/src/components/pages/Category/index.vue]: .

File: resources/frontend/src/components/pages/Category/index.vue [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+                    this.getListCategories();
+                }else{
+                    console.log(result);
```
Comment (VanTan291) [inline resources/frontend/src/components/pages/Category/index.vue:160]: remove log đi leader

**Comment by phucbienvan** [comment]
Team manh day ✌️

File: resources/frontend/src/enums/CategoryStatus.js [hunk 10]
```diff
@@ -0,0 +1,4 @@
+export default {
+    ACTIVE: 1,
+    INACTIVE: 0,
+}
```
Comment (VanTan291) [inline resources/frontend/src/enums/CategoryStatus.js:4]: .

File: app/Http/Requests/CategoryRequest.php [hunk 11]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Http\Request;
+
+class CategoryRequest extends BaseRequest
+{
+    public function rules(Request $request)
+    {dd($request->all());
```
Comment (VanTan291) [inline app/Http/Requests/CategoryRequest.php]: .

**Pull request merged**